dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -16,11 +16,30 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+# tflint-ignore: terraform_naming_convention
```

Comment:
Can you actually fix the naming, please, since this is a new resource?

Reply:
This is about the name of TF resource, not the name of the topic

Reply:
Ah I was under the impression that it had to match the name of the source topic, is that not the case?

Reply:
Nope, they are separate things and they don't need to match

Reply:
Okay, good to know. That's it fixed